data/almalinux.yml

```diff
@@ -0,0 +1,281 @@
+---
+- name: AlmaLinux-8
+  version: '8'
+  old_versions:
+    - '8.6'
```

Comment:
How do you handle if some version does not have a particular arch? Just skip the repo?

Reply:
in case if some repos doesn't exist, I skip them and save traceback in `last_error` field